src/test/java/com/google/firebase/auth/ListUsersPageTest.java

```diff
@@ -45,7 +46,7 @@
   @Test
   public void testSinglePage() throws FirebaseAuthException, IOException {
     TestUserSource source = new TestUserSource(3);
-    ListUsersPage page = new ListUsersPage.PageFactory(source).create();
+    ListUsersPage page = new UserPageFactory(source).create();
```

Comment:
Lets see if we can keep this as is, or just rename to `ListUsersPage.Factory`.

Reply:
Renamed to `ListUsersPage.Factory`.